back/lib/tasks/data_migrate/20230828_populate_initiative_posting_tips.rake

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+namespace :data_migrate do
+  block_names = %w[elaborate meaningfulTitle visualise relevantAttachments shareSocialMedia]
+  locale_file_url = 'https://raw.githubusercontent.com/CitizenLabDotCo/citizenlab/6e82b854cc8c05a0ea5ab2b5acf5b7ed3f76bf47/front/app/translations/%<locale>s.json'
```

Comment:
I think I'm missing something obvious here, but where is this file? I can't find it.

Reply:
We discussed, and this works.